src/main/java/com/onevizion/scmdb/facade/DbScriptFacade.java

```diff
@@ -72,6 +71,8 @@ public void copyRollbacksToExecDir(List<SqlScript> rollbacks) {
     public void copyRollbackToExecDir(SqlScript rollback) {
         File rollBackFile = new File(execDir.getAbsolutePath() + File.separator + rollback.getName());
         rollback.setFile(rollBackFile);
+        String text = sqlScriptDaoOra.readScriptTextById(rollback.getId());
```

Comment:
Making db calls in cycle is a bad practise, this method can be called for hundreds of rollback. 
Refactor `sqlScriptDaoOra.readScriptTextById();` to `sqlScriptDaoOra.fidTextByIds(Set<Long> ids)` and made other required modifications on code

Reply:
Done.
I did it a different way.
I've added method `DbScriptDaoOra.readMapByIds()` to read script texts from the database by ids from any list of SqlScripts and populate the texts to an original list. This method is called when rollbacks copy to exec dir.

Reply:
see my comment above